tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.robot

```diff
@@ -26,7 +26,7 @@ Remove a stopped container
     Should Be Equal As Integers  ${rc}  0
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${container}
     Should Be Equal As Integers  ${rc}  0
-    Sleep  5 seconds
+    Sleep  10 seconds
```

Comment:
Yes, that would be much better. I think we should have a general issue to cover removing sleeps and replacing with tests